src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -39,11 +39,6 @@ public virtual bool TryCreate(FunctionMetadata functionMetadata, out FunctionDes
 
             functionDescriptor = null;
 
-            if (functionMetadata.IsDisabled)
```

Comment:
This is where we were skipping it entirely before - wouldn't be added to functions list at all